src/streaming/controllers/BufferController.js

```diff
@@ -735,16 +741,11 @@ function BufferController(config) {
     }
```

Comment:
`triggerEvent(Events.SEEK_TARGET, {time: e.from, mediaType: type, streamId: streamInfo.id});`
I think we can remove the mediaType and the streamId here

Reply:
Yes of course, you're right